app/javascript/packs/weeatApp/components/restaurantTable.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Restaurant from './restaurants';
+import Map from './map';
+
+
+class RestaurantTable extends React.Component {
```

Comment:
for simple component (i.e that have no lifecycle events) you could use this syntax:

```
const SomeComponent = props => {
  return <CustomElement />
};
```

Reply:
I have changed most of the components to function on the following PR.
Are function and the style you suggested the same? Should I change to that style?